### PR TITLE
docs: Add Google OAuth example

### PR DESCRIPTION
**Pull request by aviv926**
This PR adds screenshots for Google OAuth setup.

### CODE REVIEW

**Comment by jrasm91** [comment]
I think we can include the screenshot for Google cloud, but for the immich side, let's just include the example values in plain text instead, since screenshots quickly become outdated.

**Comment by aviv926** [comment]
> I think we can include the screenshot for Google cloud, but for the immich side, let's just include the example values in plain text instead, since screenshots quickly become outdated.

Yes, it's better, I updated accordingly.

File: docs/docs/administration/oauth.md [hunk 1]
```diff
@@ -110,8 +110,45 @@ Immich has a route (`/api/oauth/mobile-redirect`) that is already configured to
...
+
+Configuration of Authorised redirect URIs (Google Console)
+
+<img src={require('./img/google-example.webp').default} width='50%' title="Authorised redirect URIs" />
+
+Configuration of OAuth in System Settings
+
+| Setting                      | Value                                                                                                  |
+| ---------------------------- | ------------------------------------------------------------------------------------------------------ |
+| Enable                       | Login with OAuth                                                                                       |
```
Comment (jrasm91) [inline docs/docs/administration/oauth.md:139]: Is this supposed to have the value of true?

Comment (aviv926) [reply docs/docs/administration/oauth.md]: A mistake, I fixed it now. 👌

**Review by alextran1502 — APPROVED** [review]

**Review by jrasm91 — APPROVED** [review]

**Pull request merged**